README.md

```diff
@@ -72,6 +72,8 @@ Several options are currently supported:
 
 `:include-patterns` and `:exclude-patterns` accept either Strings or regex literals (regex literals cannot be embedded in EDN, so use string equivalents instead).
 
+Windows users: The backslash is not recognized as a file separator, use the forward slash `/` for the file separator character in paths and patterns.
```

Comment:
Isn't the point of this PR that backslashes would automatically be replaced?

Reply:
Nevermind. I don't think I understood the PR fully. So the patterns are specified with forward-slashes, and then when we get paths back programmatically we convert those to UNIX-style as well. Gotcha